testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -76,23 +76,23 @@ public void testCopyOnWriteShowCreateTable()
             String lastCommitTimeSync = (String) onHudi().executeQuery("show TBLPROPERTIES " + tableName + " ('last_commit_time_sync')").project(2).getOnlyValue();
             Assertions.assertThat((String) onHudi().executeQuery("SHOW CREATE TABLE default." + tableName).getOnlyValue())
                     .isEqualTo(format("""
-                                    CREATE TABLE `default`.`%s` (
-                                      `_hoodie_commit_time` STRING,
-                                      `_hoodie_commit_seqno` STRING,
-                                      `_hoodie_record_key` STRING,
-                                      `_hoodie_partition_path` STRING,
-                                      `_hoodie_file_name` STRING,
-                                      `id` BIGINT,
-                                      `name` STRING,
-                                      `price` INT,
-                                      `ts` BIGINT)
+                                    CREATE TABLE default.%s (
+                                      _hoodie_commit_time STRING,
+                                      _hoodie_commit_seqno STRING,
+                                      _hoodie_record_key STRING,
+                                      _hoodie_partition_path STRING,
+                                      _hoodie_file_name STRING,
+                                      id BIGINT,
+                                      name STRING,
+                                      price INT,
+                                      ts BIGINT)
```

Comment:
was this change "required" for the upgrade?

Reply:
Yes, it comes from the image upgrade.